packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -918,14 +863,11 @@ export const Webchat = forwardRef((props, ref) => {
       }}
     >
       {!webchatState.isWebchatOpen && (
-        <div
-          onClick={event => {
-            toggleWebchat(true)
-            event.preventDefault()
-          }}
-        >
-          {triggerButton()}
-        </div>
+        <TriggerButton
+          theme={theme}
+          unreadMessages={unreadMessages}
+          webchatState={webchatState}
+        />
```

Comment:
:clap:

Reply:
With the latest refactor there is no need for a prop 🚀